examples/resources/maas_static_route/resource.tf

```diff
@@ -0,0 +1,27 @@
+resource "maas_fabric" "tf_fabric" {
+  name = "tf-fabric"
+}
+
+resource "maas_vlan" "tf_vlan" {
+  fabric = maas_fabric.tf_fabric.id
+  vid    = 14
+  name   = "tf-vlan14"
+}
+
```

Comment:
I believe static routes link between two subnets, destination and source, so it would be good to also add a destination subnet resource here, maybe on the same fabric and VLAN but with a different cidr?